packages/lit-html/src/directives/async-append.ts

```diff
@@ -125,3 +125,9 @@ class AsyncAppendDirective extends AsyncDirective {
  *     value. Useful for generating templates for each item in the iterable.
  */
 export const asyncAppend = directive(AsyncAppendDirective);
+
+/**
+ * Non-callable type of the directive class. Necessary for when a function or
+ * method returns the return type of the above directive.
+ */
```

Comment:
Here and elsewhere, this comment isn't super clear. A non-callable type is a specific thing (it's a type without a call signature), but that's not what we mean here. Consider:

```typescript
/**
 * The type of the class that powers this directive. Necessary for naming the directive's 
 * return type.
 */
```